package.json

```diff
@@ -112,8 +113,9 @@
     "test:all": "npm run test && npm run test:functional && npm run test:e2e",
     "pre:ci": "npm run lint",
     "coverage:publish": "codecov",
-    "lint": "npm run type-check && npm run lint:ts",
+    "lint": "npm run type-check && npm run lint:ts && npm run lint:lockfile",
     "lint:ts": "eslint . --ext .js,.ts",
+    "lint:lockfile": "lockfile-lint --path yarn.lock --type yarn --validate-https --allowed-hosts verdaccio npm yarn",
```

Comment:
through the CLI command arguments, there isn't a config file (do you think it needs one?).
More help on the README: https://github.com/lirantal/lockfile-lint/tree/master/packages/lockfile-lint but generally this would look something like this:

```
lockfile-lint --type <yarn|npm> --path <path-to-lockfile> --validate-https --allowed-hosts <URL-to-match>
```

so you can provide URLs of the repository such as `https://www.npmjs.org` and others via space delimited arguments. I have used built-in aliases for `verdaccio`, `npm`, `npmjs` and `yarn` already so you can just type those in and don't need the full URL for them.

Reply:
☝️ @juanpicado